README.md

```diff
@@ -1,3 +1,16 @@
 # Hello World
 
-### this is my microverse git assignment
+> Hello world is my first microverse project to establish me in git skills.
+
+
+### Built With
+
+- HTML
+- CSS
+
+### Getting Started
+
+**git skills are very important to get as they are wildely used in the real world.**
+
+## Contacts
+[https://instagram.com/mw_imma](insta- @mw_imma) | [mailto:emmanuellmhango@gmail.com](emmanuellmhango@gmail.com)
```

Comment:
- [ ] For a professional README file, please 🙏🏽 complete it ass the last code reviewer said.

> Your README file is not contentful. Please use the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse and then customize it to your project 😉